frontend/mock-backend/data/v1/runtime/mock-retry-runtime.ts

```diff
@@ -190,7 +190,7 @@ export default {
               s3: {
                 accessKeySecret: {
                   key: 'accesskey',
-                  name: 'mlpipeline-minio-artifact',
+                  name: 'mlpipeline-object-storage-artifact',
```

Comment:
For backwards compatibility the name of the secret and key must stay the same for now.